dotnet-core-music-linux/azuredeploy.json

```diff
@@ -42,9 +35,17 @@
       "metadata": {
         "description": "Globally unique DNS Name for the Public IP used to access the Virtual Machine."
       }
+    },
+    "fileUris": {
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc
for how to handle artifacts